.circleci/config.yml

```diff
@@ -21,6 +21,14 @@ aliases:
         only: /.*/
       branches:
         only: master
+  - &only_master_and_create_dev
+    filters:
+      tags:
+        only: /.*/
+      branches:
+        only:
+        - master
+        - p4-1200-create-dev-env
```

Comment:
I am not CI expert, we are putting this here so we do not need to merge it on master?